tests/performance_tests/README.md

```diff
@@ -0,0 +1,605 @@
+# Performance Harness Tests
+
+The `performance_test.py` script performs a binary search of EOS Token Transfers Per Second (TPS) range at configurable low test duration scoring each individual test scenario to find where TPS seems to be topping out. It does this by iteratively configuring and running `performance_test_basic.py` tests and analyzing the output to determine a success metric to continue the search.  The script then proceeds to conduct an additional search with longer duration test runs at a narrowed TPS window around the discovered maximum TPS throughput. Finally it produces a report on the entire performance run, summarizing each individual test scenario, results, and full report details on the tests when maximum TPS was achieved ([Performance Test Report](#performance-test))
```

Comment:
The first sentence appears to be the victim of too many piecemeal revisions.  It could use a comprehensive rewrite in less colloquial English.

Reply:
Addressed: [b641304](https://github.com/AntelopeIO/leap/pull/352/commits/b641304b5c62607aaac45e49a9d1de9cf701bba6)